integration/hurl/tests_ok/http_version_option.hurl

```diff
@@ -13,34 +22,94 @@ GET http://localhost:8000/http_version/10
 http1.0: true
 HTTP/1.1 200
 
+
+GET http://localhost:8000/http_version/10
+[Options]
+http1.0: true
+HTTP/1.1 200
```

Comment:
`HTTP/1.1 200` should be `HTTP 200` (we move the implicit check on an explicit ones)